core/digital-twin-lib/src/utils/PermissionUtils.ts

```diff
@@ -139,33 +139,6 @@ export default class PermissionUtils {
     return perms;
   }
 
-  /**
-   * Clones whole permissions Object and removes all access rights.
-   *
-   * @param userPermissions
-   * - permissions
-   */
-  static convertToPristine(userPermissions: PermissionsContainer): PermissionsContainer {
```

Comment:
Was used for the old sharing handling, because multiple user were loaded and the `new` field was generated using that.

Is now handled with the following :

```ts
  if (!containerPermObj.permissions) {
      containerPermObj.permissions = PermissionUtils
        .createPermissionsObject({ read: [], readWrite: [] }, properties);
    }
```